.eslintrc

```diff
@@ -16,10 +16,6 @@
     "navigator": false,
     "unescape": false,
     "window": false,
-    "$": true,
-    "_": true,
-    "__": true,
-    "n__": true,
     "d3": true
```

Comment:
do we need to keep d3?

Reply:
+1 for removing d3.
We can put comments where we do use it.

Actually, can't we remove the whole `globals` field?

Reply:
ping @amirfefer - any updates on this?